test/clonevolume_tester.py

```diff
@@ -0,0 +1,262 @@
+import mock
+import testtools
+
+import fake_3par_data as data
+import createvolume_tester as createvolume
+from hpe3parclient import exceptions
+
+
+# Variation of CreateVolumeUnitTest. Nothing specific to do here
+class CloneVolumeUnitTest(createvolume.CreateVolumeUnitTest):
+    pass
+
+
+# This exercises online-copy path
+class TestCloneDefault(CloneVolumeUnitTest):
```

Comment:
can you comment this test case as  To Do , here assert calls for checking the necessary calls are not present.
This looks to be incomplete test case

Reply:
Actually CloneVolumeUnitTest is the base class for all the clone test cases and it takes all its implementation from its parent class CreateVolumeUnitTest.